src/app/malaria-101/activity-3/activity-3.component.ts

```diff
@@ -23,8 +24,11 @@ export class OddOneOutComponent implements OnInit {
     private _subscription;
     private _data;
     private _status: object = {stage: 2, activity: 3};
+    private readonly POINTS_PER_CORRECT_ANSWER = 10;
+    private readonly QUESTION_NUMBER_FIVE = 5;
```

Comment:
Isn't it better to change the variable name to LAST_QUESTION or something

Reply:
Done.